scripts/seed.py

```diff
@@ -0,0 +1,210 @@
+    team = ChallengeHostTeam.objects.create(
+        team_name=team_name,
+        created_by=user,
+    )
+    print "Challenge Host Team created with \n team_name: %s \n created_by: %s" % (team_name, user.username)
+    ChallengeHost.objects.create(user=user, team_name=team, status="Self", permissions="Admin")
+    print "Challenge Host created with \n user: %s \n team_name: %s" % (user.username, team_name)
+    return team
+
+
+def create_challenges(number_of_challenges=3, host_team=None):
+    fake = Factory.create()
+    for i in xrange(number_of_challenges):
+        if (i % 3 == 0):
+            create_challenge_object("%s Challenge" % (
+                fake.first_name()),
+                timezone.now() - timedelta(days=100),
+                timezone.now() + timedelta(days=500),
+                host_team
+            )
+        elif (i % 3 == 1):
+            create_challenge_object("%s Challenge" % (
+                fake.first_name()),
+                timezone.now() - timedelta(days=500),
+                timezone.now() - timedelta(days=100),
+                host_team
+            )
+        elif (i % 3 == 2):
+            create_challenge_object("%s Challenge" % (
+                fake.first_name()),
+                timezone.now() + timedelta(days=100),
+                timezone.now() + timedelta(days=500),
+                host_team
+            )
+
+
+def create_challenge_object(title, start_date, end_date, host_team):
+    fake = Factory.create()
+    evaluation_script = open(os.path.join(settings.BASE_DIR, 'examples', 'example1', 'string_matching.zip'), 'rb')
+    Challenge.objects.create(
+        title=title,
+        short_description=fake.paragraph(),
+        description=fake.paragraph(),
+        terms_and_conditions=fake.paragraph(),
+        submission_guidelines=fake.paragraph(),
+        evaluation_details=fake.paragraph(),
+        evaluation_script=SimpleUploadedFile(evaluation_script.name, evaluation_script.read()),
+        creator=host_team,
+        published=True,
+        enable_forum=True,
+        anonymous_leaderboard=False,
+        start_date=start_date,
+        end_date=end_date,
+    )
+    print "Challenge created with \n title: %s \n creator: %s \n start_date: %s \n end_date: %s"\
+          % (title, host_team.team_name, start_date, end_date)
+
+
+def create_challenge_phases(number_of_phases=1):
+    fake = Factory.create()
+    for i in range(number_of_phases):
+        name = "%s Phase" % (fake.first_name())
+        ChallengePhase.objects.create(
+            name=name,
+            description=fake.paragraph(),
+            leaderboard_public=True,
+            is_public=True,
+            start_date=challenge.start_date,
+            end_date=challenge.end_date,
+            challenge=challenge,
+            test_annotation=SimpleUploadedFile(fake.file_name(extension="txt"),
+                                               "1\n2\n3\n4\n5\n6\n7\n8\n9\n10", content_type="text/plain"),
```

Comment:
For the string `1\n`, you can declare it once and then re-use it here. range and join can be used to create it in pythonic way.

Also is this really used somewhere or a dummy string where in you have just placed some number. In case if it is a dummy string can we make it something more meaningful.